docs/migrations/ibc-migration-042.md

```diff
@@ -0,0 +1,54 @@
+# Migrating to ibc-go
+
+This file contains information on how to migrate from the IBC module contained in the SDK 0.41.x line to the IBC module in the ibc-go repository based on the 0.42 SDK version. 
+
+## Import Changes
+
+The most obvious changes is import name changes. We need to change:
+- applications -> apps
+- cosmos-sdk/x/ibc -> ibc-go
+
+On my GNU/Linux based machine I used the following commands, executed in order:
+
+`grep -RiIl 'cosmos-sdk\/x\/ibc\/applications' | xargs sed -i 's/cosmos-sdk\/x\/ibc\/applications/ibc-go\/apps/g'`
```

Comment:
```suggestion
`grep -RiIl 'cosmos-sdk\/x\/ibc\/applications' | xargs sed -i 's/cosmos-sdk\/x\/ibc\/applications\/ibc-go\/apps/g'`
```

Reply:
No they are correct. I will add a link to the [source](https://www.internalpointers.com/post/linux-find-and-replace-text-multiple-files)

Adding the `\` causes that to be interpreted as one long path `cosmos-sdk/x/ibc/applications/ibc-go/apps` as opposed to the 
search: `cosmos-sdk/x/ibc/applications`
replace: `ibc-go/apps`